controllers/userController.js

```diff
@@ -33,9 +34,18 @@ const signUp = catchAsync(async (req, res) => {
 
 })
 
+const getLikeList = catchAsync(async (req, res) => {
+  const { userId } = req.body;
+    
+    const likes = await userService.getLikeList(userId);
+    
+    res.status(201).json({ likes : likes});
```

Comment:
```suggestion
    res.status(200).json({ likes });
```
- 201 status code는 Created에 대한 status code입니다! 200으로 수정해 주세요~
- likes만 써도 같은 역할을 할 것 같아요!

Reply:
수정완료했습니다.